kinto/core/storage/postgresql/__init__.py

```diff
@@ -243,6 +243,9 @@ def create(self, collection_id, parent_id, record, id_generator=None,
         VALUES (:object_id, :parent_id,
                 :collection_id, (:data)::JSONB,
                 from_epoch(:last_modified))
+        ON CONFLICT (id, parent_id, collection_id) DO UPDATE
```

Comment:
Ok it makes sense. Let's find out how we can create a test to reproduce that. I guess we had one with loadtest simulation before.